src/main/groovy/liquibase/harness/util/TestUtils.groovy

```diff
@@ -46,6 +47,9 @@ class TestUtils {
         try {
             commandScope.execute()
         } catch (Exception exception) {
+            if (exception instanceof CommandExecutionException && exception.toString().contains("is not available in SQL output mode")) {
```

Comment:
not obvious why we are doing this, would be nice to create separate method for handling it (as check may expand in future) or at least add comment

Reply:
fixed